src/index.html

```diff
@@ -54,7 +55,7 @@
 
     <div id="search" class="container">
       <div class="card-deck mb-4 pt-3">
-        <form class="card" onsubmit="return false">
+        <form class="card">
```

Comment:
Note-to-self: the reason to remove the `return false` JavaScript snippet from this and another `<form>` element is that otherwise, these are considered as script elements according to the Content-Security-Policy.  It doesn't seem sensible to add another hash to the policy purely to allow `return false`; instead it's better to move the logic (48229e3afe37f78c5938fa04b5c9fd0e539eb628) into one of the existing application modules, which already has a hash.